src/main/webapp/content/scss/global.scss

```diff
@@ -827,14 +826,12 @@ Course Info Bar
     }
 }
 
-.warning-modal-window .modal-dialog {
-    width: 50%;
+.diff-view-modal .modal-dialog {
+    max-width: 80vw;
 }
```

Comment:
I had to move this declaration here to ensure that we wouldn't encounter visual glitches when leaving the page